qosf.org/qc_mentorship.md

```diff
@@ -42,23 +42,28 @@ The idea for this program comes from the fact that many people from diverse back
 
 By pairing people with mentors we want to help them overcome these hurdles and create interesting projects, which will, in turn, will help others.
 
-Learn more about the story behind this program (and its predecessor) on [Michał Stęchły’s blog](https://www.mustythoughts.com/QC-Mentorship-Program-backstory.html). 
+One of the program’s organizers (Michał Stęchły) shared some "behind the scenes" information about the program on his blog. If you're interested, you can read the story behind the [program](https://www.mustythoughts.com/qc-mentorship-program-backstory) or about how the application process looks like from our [perspective](https://www.mustythoughts.com/qosf-mentorship-application.html).
```

Comment:
I would change the links to work on the whole phrases [the story behind the program]  and [how the application .. perspective]